cms/djangoapps/contentstore/rest_api/v1/serializers/course_waffle_flags.py

```diff
@@ -114,9 +114,11 @@ def get_use_new_video_uploads_page(self, obj):
     def get_use_new_course_outline_page(self, obj):
         """
         Method to get the use_new_course_outline_page switch
+
+        Always true, because the switch is being removed an the new experience
+        should alawys be on.
```

Comment:
```suggestion
        Always true, because the switch is being removed and the new experience
        should always be on. This function will be removed in
        https://github.com/openedx/edx-platform/issues/37497
```